added_mass/proposal.md

```diff
@@ -0,0 +1,255 @@
+* $\mu_{35}$ : added mass in the Z axis due to angular acceleration about the X direction
+* $\mu_{36}$ : added mass in the Z axis due to angular acceleration about the X direction
+* $\mu_{41} = \mu_{14}$ : added mass moment about the X axis due to linear acceleration in the X direction, and vice-versa
+* $\mu_{42} = \mu_{24}$ : added mass moment about the X axis due to linear acceleration in the Y direction, and vice-versa
+* $\mu_{43} = \mu_{34}$ : added mass moment about the X axis due to linear acceleration in the Z direction
+* $\mu_{44}$ : added mass moment about the X axis due to angular acceleration about the X direction
+* $\mu_{45}$ : added mass moment about the X axis due to angular acceleration about the X direction
+* $\mu_{46}$ : added mass moment about the X axis due to angular acceleration about the X direction
+* $\mu_{51} = \mu_{15}$ : added mass moment about the Y axis due to linear acceleration in the X direction, and vice-versa
+* $\mu_{52} = \mu_{25}$ : added mass moment about the Y axis due to linear acceleration in the Y direction, and vice-versa
+* $\mu_{53} = \mu_{35}$ : added mass moment about the Y axis due to linear acceleration in the Z direction, and vice-versa
+* $\mu_{54} = \mu_{45}$ : added mass moment about the Y axis due to angular acceleration about the X direction, and vice-versa
+* $\mu_{55}$ : added mass moment about the Y axis due to angular acceleration about the X direction
+* $\mu_{56}$ : added mass moment about the Y axis due to angular acceleration about the X direction
+* $\mu_{61} = \mu_{16}$ : added mass moment about the Z axis due to linear acceleration in the X direction, and vice-versa
+* $\mu_{62} = \mu_{26}$ : added mass moment about the Z axis due to linear acceleration in the Y direction, and vice-versa
+* $\mu_{63} = \mu_{36}$ : added mass moment about the Z axis due to linear acceleration in the Z direction, and vice-versa
+* $\mu_{64} = \mu_{46}$ : added mass moment about the Z axis due to angular acceleration about the X direction, and vice-versa
+* $\mu_{65} =\mu_{56}$ : added mass moment about the Z axis due to angular acceleration about the X direction, and vice-versa
+* $\mu_{66}$ : added mass moment about the Z axis due to angular acceleration about the Z direction
+
+&nbsp;
+
+&nbsp;
+
+## Proposed changes
+
+The [`//link/inertial`](http://sdformat.org/spec?ver=1.9&elem=link#link_inertial) term
+in SDF currenly accepts elements to describe the m matrix (mass and moments of inertia),
+as well as a center of mass pose. This proposal suggests exposing each of the 21 elements
+for the mass matrix in addition to that. All added mass elements are optional and default
+to zero if unset. This preserves the behaviour for links that don't have those terms.
+
+### XML spec
+
+A new `<added_mass>` element will be added under `//link/inertial/`. It will contain each of the
+21 matrix elements started by `n` and followed by its index.
```

Comment:
I like the change of elements, it also has the advantage of mirroring the notation in `inertia`.

Reply:
For the `hydrodynamic` part, one concern is that these terms also apply to other objects submerged in fluids other than water, like air. Not sure if something like `fluid_added_mass` makes enough sense?

Reply:
All of this looks fine for our purposes here.  
- For me, \mu is the most natural and general, but the letter notation is fine too, if a bit non-standard.  But mirroring the inertia tensor sounds good.
- In the fluid dynamics world, air is most certainly understood to be a fluid so that distinction doesn't keep me up at night at all.  And you are right, in the case of a balloon in air, the added mass is a major contribution to inertia...   
-  Similarly, in my experience "added mass" is a very common noun so I would hesitate to insert the world fluid in the middle.  Actually, I would eliminate the inclusion of "fluid" entirely and simply refer to these quantities as "added mass", the problem is the fluid is causing the added inertia, so people usually refer to the bodies added mass, not the fluid.  Ugh, semantics.

Reply:
I adopted the term "fluid added mass" throughout the proposal in [0c24fbc](https://github.com/gazebosim/sdf_tutorials/pull/76/commits/0c24fbcf822d1ce879129ba9ed6f96a0d3f79713). I believe all items in this comment have been addressed.